packages/contentstack-import/src/import/modules-js/entries.js

```diff
@@ -38,7 +38,7 @@ module.exports = class ImportEntries {
   ctPath;
   lPath;
   importConcurrency;
-  skipFiles = ['__master.json', '__priority.json', 'schema.json'];
+  skipFiles = ['__master.json', '__priority.json', 'schema.json','.DS_Store'];
```

Comment:
@cs-raj Are there any similar TS changes required in the TS version of entries.js?

Reply:
No @netrajpatel. This is already handled there

Reply:
Correction from my end: I connected with Shafeqq and Learnt that even though we have skipFiles in content types. In both entries and content type we are reading from schema.json due to which the issue of .DS_Store won't arise